src/AWS.Deploy.Orchestration/ServiceHandlers/AWSElasticBeanstalkHandler.cs

```diff
@@ -37,6 +82,39 @@ public AWSElasticBeanstalkHandler(IAWSClientFactory awsClientFactory, IOrchestra
             _optionSettingHandler = optionSettingHandler;
         }
 
+        /// <summary>
+        /// Elastic Beanstalk Windows deployments require a manifest file to be included with the application to be deployed.
+        /// This function creates the windows manifest file and adds it to the zip file that will be deployed to Beanstalk.
+        /// </summary>
+        public void SetupWindowsDeploymentManifest(Recommendation recommendation, string dotnetZipFilePath)
+        {
+            const string MANIFEST_FILENAME = "aws-windows-deployment-manifest.json";
+            var windowsManifest = new ElasticBeanstalkWindowsManifest();
+
+            var iisWebSiteOptionSetting = _optionSettingHandler.GetOptionSetting(recommendation, "IISWebSite");
+            var iisAppPathOptionSetting = _optionSettingHandler.GetOptionSetting(recommendation, "IISAppPath");
+
+            var iisWebSiteValue = _optionSettingHandler.GetOptionSettingValue<string>(recommendation, iisWebSiteOptionSetting);
+            var iisAppPathValue = _optionSettingHandler.GetOptionSettingValue<string>(recommendation, iisAppPathOptionSetting);
+
+            var iisWebSite = !string.IsNullOrEmpty(iisWebSiteValue) ? iisWebSiteValue : "Default Web Site";
+            var iisAppPath = !string.IsNullOrEmpty(iisAppPathValue) ? iisAppPathValue : "/";
```

Comment:
It is possible in the legacy wizard they set the app path to `/foobar` then when we switch over to the new wizard we are hard coding to `/`. We need to make sure there is a UI setting for app path and that value gets funneled into here.

Reply:
Fixed